doc/primitives.md

```diff
@@ -0,0 +1,268 @@
+Embedded graphics examples
```

Comment:
Is this an old version/name of the `drawing-examples.md` file?

Reply:
Oops, yes that shouldn't be in here at all.